asa_parser/tests/test_parser.py

```diff
@@ -65,23 +67,9 @@ def test_show_process_cpu_hog(self):
 
 
         
-    def test_startup_config_errors(self):
```

Comment:
It looks like you are removing some code that shouldn't be removed.  Probably you are using an older version of dev.  It looks like your fork it 8 commits behind currently.  You'll need to re-sync your fork to latest and then make sure this code is not being removed.  If it is, you probably merged incorrectly.